backend/hardhat.config.ts

```diff
@@ -94,15 +95,16 @@ task('setMessage')
   })
 
 // Hardhat Node and sapphire-dev test mnemonic.
-const TEST_HDWALLET = {
-  mnemonic: 'test test test test test test test test test test test junk',
+const TEST_HDWALLET: HDAccountsUserConfig = {
+  mnemonic: "test test test test test test test test test test test junk",
   path: "m/44'/60'/0'/0",
   initialIndex: 0,
   count: 20,
-  passphrase: '',
-}
+  passphrase: "",
+};
+const firstPrivateKey = Wallet.fromPhrase(TEST_HDWALLET.mnemonic).privateKey;
 
-const accounts = process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : TEST_HDWALLET
+const accounts = process.env.PRIVATE_KEY ? [process.env.PRIVATE_KEY] : [firstPrivateKey];
```

Comment:
Is there a reason for having a single private key? There may be some tests requiring two users for example `accounts[0]` and `accounts[1]`. I would try to keep the existing behavior.